src/libstore/file-hash.cc

```diff
@@ -0,0 +1,53 @@
+#include "file-hash.hh"
```

Comment:
might want to call this "content-address.cc/.hh" since that's what it mostly provides. otherwise it's a bit confusing what the difference is between this and all of the other hash functions in hash.cc.

Reply:
Moved, thanks